crates/torii/core/src/engine.rs

```diff
@@ -627,14 +691,30 @@ impl<P: Provider + Send + Sync + std::fmt::Debug> Engine<P> {
             return Ok(());
         };
 
-        // if processor.validate(event) {
-        if let Err(e) = processor
-            .process(&self.world, &mut self.db, block_number, block_timestamp, event_id, event)
-            .await
-        {
-            error!(target: LOG_TARGET, event_name = processor.event_key(), error = %e, "Processing event.");
+        let task_identifier = match processor.event_key().as_str() {
+            "StoreSetRecord" | "StoreUpdateRecord" | "StoreUpdateMember" | "StoreDelRecord" => {
+                poseidon_hash_many(&[event.data[0], event.data[1]])
```

Comment:
Poseidon looks heavy for having a task id, do you think we could switch to something less intensive?

Reply:
Let's tackle this in an other PR and see how we could get something performant here, since we're executing this action for every single event.
Working with bytes and updating a hasher with both felts should be the most interesting for us here, as lamdaworks is also working at bytes level.

Reply:
Just addressed this, we're using the rust default hasher that uses the felts directly since they implement Hash. This should do it.